### ISSUE TITLE
Can not root

### ISSUE THREAD

**Issue by 4119ranger** [fpv-wtf/wtfos-configurator#313]
### Device

Goggles V1

### Trace ID (bototm left)

7cc1034f-a776-45d1-9100-349a302b88ee

### Describe the issue

I have used 3 batteries and even though I had already rooted the goggles months ago Atfer instsalling ELRS I could no longer record video on the air unit so i went back to start over again. i could not connect except to root and then it always fails here.

00:03 Attempting Step 1...

00:17 - Found Device: GL150, Version: 01.00.0606

00:17 Step 1 - Success! Rebooting...

01:19 Aborted! Device did not reboot in time, please power cycle and try again.

**Comment by stylesuxx** [comment]
Sorry, I am a bit confused by your error description. What does recording videos on the air unit have to do with rooting the goggle, and where does ELRS come into play here?

I am assuming we are talking about the Goggles failing to root?

Can you please tell me which version of windows you are on?

**Comment by 4119ranger** [comment]
I had originally used WTF on my goggles and air unit and they were
performing well. I went to extend the range and used ELRS. After that I
noticed I was no longer recording video on my air unit. I have no idea why
however I thought I would mention it so I went back to check the WTF and
found I could no longer use the WTF on the goggles and needed to root the
goggles. The air unit seems to have the wtf still accessible just not the
goggles.  However i keep getting the error it starts to root then gets to
the first reboot and just stalls till time out. Any help would be great
Thanks for getting back to me.
Chuck

**Comment by stylesuxx** [comment]
OK, so we are talking about the googles - they might have switched slot. You still did not tell me whith version of windows you are on, also telll me which browser you are using. I've heard from people with similar issues on older versions of windows, they would need to power cycle manually when the google was saying it is rebooting.

**Comment by 4119ranger** [comment]
Hi Chris
 I am using Windows 10  and I used Vivaldi then I tried Google Chrome.
Neither worked for the goggles which are the first model DJI put out, same
as the air unit.  I did try to intervene on the reboot one time and did not
seem to help. After attempting the access multiple times I went back to DJI
Assistant and reflashed the goggles to their original 6006 version then
tried again with the same results.
Chuck

**Comment by stylesuxx** [comment]
Hmm, windows 10 should be fine. Are you sure that you are not running any other application that might claim the  serial device? Cure for example likes to do that.

I would ask you to put this  `chrome://device-log/` in chromes URL bar, clear the log, then connect the goggles, refresh the log and post the output.

**Comment by 4119ranger** [comment]
Chris, thank you so much for your help after I wrote to you last night I
installed my diag, hard drive with a virgin win 10 hard drive.  Actually
when I answered your reply to my email this morning I had not even loaded
chrome yet on it. Anyway I went to get the logs for you and it worked on
the first try. It is apparent the problem was as you said another app was
using resources WTF needed.

Thanks again
Chuck
